src/sfizz/Synth.cpp

```diff
@@ -962,3 +966,11 @@ void sfz::Synth::disableLogging() noexcept
 {
     resources.logger.disableLogging();
 }
+
+void sfz::Synth::allSoundOff() noexcept
+{
+    for (auto &voice: voices)
+        voice->reset();
+    for (auto& effectBus: effectBuses)
+        effectBus->clear();
+}
```

Comment:
Should this method not have the `AtomicDisabler` like others?

Reply:
Good point.